components/unzipped/dashboard/Kanban/KanbanBoard.js

```diff
@@ -0,0 +1,327 @@
+  padding: grid * 2,
+  margin: `0 0 ${grid}px 0`,
+
+  // change background colour if dragging
+  background: isDragging ? "lightgreen" : "grey",
+
+  // styles we need to apply on draggables
+  ...draggableStyle
+});
+const getListStyle = (isDraggingOver) => ({
+  background: isDraggingOver ? "lightblue" : "lightgrey",
+  padding: grid,
+  width: 250
+});
+
+// Global Style
+const GlobalStyle = createGlobalStyle`
+  body {
+    font-family: 'Arial', sans-serif;
+    margin: 0;
+    padding: 0;
+  }
+`;
+
+// Styled Components
+const Container = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  width: 100%;
+`;
+
+const Board = styled.div`
+  display: flex;
+  justify-content: space-around;
+  align-items: flex-start;
+  background-color: #ffffff;
+  border-radius: 8px;
+  padding: 0px 20px;
+  width: 100%;
+  height: calc(100vh - 244px); 
+  overflow: scroll;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+
+  ::-webkit-scrollbar {
+    width: 5px;
+  }
+
+  ::-webkit-scrollbar-track {
+    background: #f1f1f1;
+
+  ::-webkit-scrollbar-thumb {
+    background: #888;
+    border-radius: 5px; 
+  }
+
+  ::-webkit-scrollbar-thumb:hover {
+    background: #555;
+  }
+`;
+
+const ColumnContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 0 10px;
+  width: 300px;
+`;
+
+const ColumnTitle = styled.h2`
+  font-size: 24px;
+  color: #000000;
+  margin: 15px 0px;
+  padding: 0px;
+  text-align: center;
+`;
+
+const DroppableArea = styled.div`
+  background-color: ${(props) => (props.isDraggingOver ? '#34D9B3' : '#e0e4e7')};
+  padding: 8px;
+  width: 100%;
+  min-height: ${(props) => props.minHeight}px; /* Dynamic height */
+  border-radius: 8px;
+`;
+
+// Data and Functions
+const itemsFromBackend = [
+  {
+    id: uuid(),
+    ticketNumber: 101,
+    title: "Create full screen mode for task list page",
+    description: "This is the description for the first task.",
+    dueDate: "2024-05-20"
+  },
+
+  {
+    id: uuid(),
+    ticketNumber: 102,
+    title: "Make changes to the signup flow",
+    description: "This is the description for the second task.",
+    dueDate: "2024-05-21"
+  },
+
+  {
+    id: uuid(),
+    ticketNumber: 103,
+    title: "Verify identity not working in staging",
+    description: "This is the description for the third task.",
+    dueDate: "2024-05-22"
+  },
+
+  {
+    id: uuid(),
+    ticketNumber: 104,
+    title: "Create security alert or unusual activity email",
+    description: "This is the description for the fourth task.",
+    dueDate: "2024-05-23"
+  },
+
+  {
+    id: uuid(),
+    ticketNumber: 105,
+    title: "Create user successfully Hired email for (client)",
+    description: "This is the description for the fifth task.",
+    dueDate: "2024-05-24"
+  }
+];
+
+const columnsFromBackend = {
+  [uuid()]: {
+    name: "New",
+    items: itemsFromBackend
+  },
+  [uuid()]: {
+    name: "To do",
```

Comment:
These items (in progress, done, etc.) Should not be hard coded. They are the tags that are associated with the project. This way a project owner can create and remove columns.